defaults/main.yml

```diff
@@ -12,20 +12,22 @@ docker_restart_handler_state: restarted
 # Docker Compose options.
 docker_install_compose: true
 docker_compose_version: "1.26.0"
+docker_compose_url: https://github.com/docker/compose/releases/download/{{ docker_compose_version }}/docker-compose-Linux-x86_64
 docker_compose_path: /usr/local/bin/docker-compose
 
 # Used only for Debian/Ubuntu. Switch 'stable' to 'nightly' if needed.
+docker_url: https://download.docker.com/linux
```

Comment:
Can you name this `docker_repo_url` or maybe `docker_download_url`? I'm just thinking long term if there is some other feature that requires a download from Docker that uses a different URL, I don't want the more generic `docker_url` taken.

Reply:
Good idea. Have amended this commit to use `docker_repo_url` instead.